src/styles/typography.js

```diff
@@ -0,0 +1,63 @@
+import { createGlobalStyle } from "styled-components"
+
+export const primaryFont = "GT America"
```

Comment:
Heh, I think you were mistaken here (no doubt due to the disorganization of the repo) but we're not intending to use any proprietary fonts here - just the `system-ui`.

Reply:
Ah okay, gotcha. I'll change that up now.